### PR TITLE
[CHORE] House Cleaning

### LINKED ISSUE #381: [CHORE] House Cleaning
## Issue Overview

A bunch of tasks to clean up the codebase
- Remove `.gitmodules` file
- Check for `CONTRIBUTING.md` and `README.md` to ensure they are up to date

### PR DESCRIPTION
**Pull request by BrunoAmbricca**
# Task name here

Fixes #381 

## Types of change

- [ ] Feature
- [ ] Bug
- [x] Enhancement

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,15 +61,15 @@ To ensure the proper functioning of the tests on scaffold-stark, your Starknet F
...
-If your Starknet Foundry version is not `0.34.0`, you need to install it.
+If your Starknet Foundry version is not `0.35.1`, you need to install it.
 
-- Install Starknet Foundry `0.34.0` via `asdf` ([instructions](https://foundry-rs.github.io/starknet-foundry/getting-started/installation.html#installation-via-asdf)).
+- Install Starknet Foundry `0.35.1` via `asdf` ([instructions](https://foundry-rs.github.io/starknet-foundry/getting-started/installation.html#installation-via-asdf)).
 
 ## Compatible versions
 
 - Starknet-devnet - v0.2.3
 - Scarb - v2.9.2
```
Comment (metalboyrick) [inline README.md:71]: this version correct? @Nadai2010

**Review by metalboyrick — COMMENTED** [review]
let's have @Nadai2010 check for the versions, i think so far looks good

File: README.md (hunk 1 above)
Comment (Nadai2010) [reply README.md:71]: Hi @BrunoAmbricca the snfoundry version is 0.34, leave it as it was.

Please check all faults in Contribution and Readme or possible improvements, thank you

**Review by Nadai2010 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BrunoAmbricca) [reply README.md]: Hello @Nadai2010. I read the contribution and readme files and didn´t find any faults to improve. Let me know if a miss anything.

**Review by Nadai2010 — APPROVED** [review]

**Pull request merged**